hive/configs/marlgrid/dqn.yml

```diff
@@ -4,8 +4,8 @@ train_steps: 500000
 test_frequency: 1000
 test_num_episodes: 10
 self_play: False
-num_agents: &num_agents 2
-stack_size: &stack_size 2
+num_agents: &num_agents 3
```

Comment:
1. How is this related to the focus of the PR?
2. If this change in the config should be pushed to dev, where is the link pointing to the generated results?
3. Please provide further explanation for these changes.

Reply:
1. There was a bug I guess in the config before because the same environment defined in - https://github.com/chandar-lab/RLHive/blob/dev/hive/envs/marlgrid/ma_envs/__init__.py, had num_agents 3 and that task doesn't run with num_agents 2. But yeah its not specific to this PR.
2. Since it was an error in the config I have not reproduced these results.